encoding/message.go

```diff
@@ -4,6 +4,14 @@
 // By Rene Nayman
 package encoding
 
+type Protocol string
+
+const (
+	RPC = "RPC"
```

Comment:
```suggestion
	RPC Protocol = "RPC"
```

Should be this for all subsequent lines too. Having one type hint only works when using iota for integer types.